formatTest/unit_tests/expected_output/syntax.re

```diff
@@ -57,59 +57,68 @@ type myRecordType = {
 };
 
 type firstNamedArgShouldBeGroupedInParens =
-  (:first int => int, :second int) => int;
+  (:first: int => int, :second: int) => int;
 
 type allParensCanBeRemoved =
-  (:first int, :second int, :third int) => int;
+  (:first: int, :second: int, :third: int) => int;
 
 type firstTwoShouldBeGroupedAndFirstThree =
-  (:first (int => int) => int) => int;
+  (:first: (int => int) => int) => int;
 
 /* Same thing now, but with type constructors instead of int */
 type firstNamedArgShouldBeGroupedInParens =
   (
-    :first list(int) => list(int),
-    :second list(int)
+    :first: list(int) => list(int),
+    :second: list(int)
   ) =>
   list(int);
 
 type allParensCanBeRemoved =
   (
-    :first list(int),
-    :second list(int),
-    :third list(int)
+    :first: list(int),
+    :second: list(int),
+    :third: list(int)
   ) =>
   list(int);
 
 type firstTwoShouldBeGroupedAndFirstThree =
   (
-    :first (list(int) => list(int)) => list(int)
+    :first: (list(int) => list(int)) => list(int)
   ) =>
   list(int);
 
 type firstNamedArgShouldBeGroupedInParens =
-  (:first int => int?, :second list(int)?) => int;
+  (:first: int => int=?, :second: list(int)=?) =>
+  int;
 
 /* The arrow necessitates parens around the next two args. The ? isn't what
  * makes the parens necessary. */
 type firstNamedArgShouldBeGroupedInParensAndSecondNamedArg =
-  (:first int => int?, :second int => int?) =>
+  (
+    :first: int => int=?,
```

Comment:
Is there an easy way to avoid the newline here?
```
type blablablablablablablabla = 
  (:firstVeryLongNameHere: int => int=?, 
   :secondOtherVeryLongNameBecauseWeNeedToWrap: int => int=?) => int;
```
Something like that?

Not super attached though, don't worry about it if it'll take more than like 10min.

Reply:
You probably only want this in type signatures?

Reply:
Mmmh true. I don’t think it’s worth having the type wrap differently than the definition or calls, so nvm forget it ;)
Great job dude